packages/contracts/scripts/generate-markdown.js

```diff
@@ -37,19 +38,15 @@ const networks = {
 
 ### Predeploy contracts:
 |Contract|Address|
-|--|--|
-|OVM_ETH: | \`0x4200000000000000000000000000000000000006\`
-|OVM_L2StandardBridge: | \`0x4200000000000000000000000000000000000010\`
-|OVM_L2CrossDomainMessenger: | \`0x4200000000000000000000000000000000000007\`
-|OVM_L2ToL1MessagePasser: | \`0x4200000000000000000000000000000000000000\`
-|OVM_L1MessageSender: | \`0x4200000000000000000000000000000000000001\`
-|OVM_DeployerWhitelist: | \`0x4200000000000000000000000000000000000002\`
-|OVM_ECDSAContractAccount: | \`0x4200000000000000000000000000000000000003\`
-|OVM_SequencerEntrypoint: | \`0x4200000000000000000000000000000000000005\`
-|Lib_AddressManager: | \`0x4200000000000000000000000000000000000008\`
-|ERC1820Registry: | \`0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24\`
+|--|--|\n`
 
----
+  for (const predeploy in predeploys) {
+    if (predeploys.hasOwnProperty(predeploy)) {
+      md += `|${predeploy}: |${predeploys[predeploy]}\n`
```

Comment:
I can also add in a link to the predeploy contract on `optimistic.etherscan.io` to bring this in line with the other contracts.

Reply:
Having a link there would be great, especially if we got the contracts verified on optimistic.etherscan.io. The addresses are the same across networks. Would it be overkill to link both kovan and mainnet?

https://kovan-optimistic.etherscan.io/
https://optimistic.etherscan.io/

Reply:
Hmmm, now that I'm thinking about it: maybe we /do/ want to have a separate predeploys section for each network? I assume there will be points in time where Kovan has X set of predeploys while Mainnet has Y.

Reply:
That sounds good to me! I'll try to get the formatting more in line with the `Layer 1` section. Do we also want to define a set of predeploys for the `Goerli` network? I'll probably DRY things up a bit too while I'm tinkering.

Reply:
Yeah I think it would be worth adding for Goerli although we don't have an explorer for it to link to.